pkgs/by-name/th/thrift-ls/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "thrift-ls";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "joyme123";
+    repo = "thrift-ls";
+    rev = "v${version}";
+    hash = "sha256-ywrb1pOpqHnVhb9MA2cgkF5croA9iqtvLxvVmeloOpE=";
+  };
+
+  vendorHash = "sha256-YoZ2dku84065Ygh9XU6dOwmCkuwX0r8a0Oo8c1HPsS4=";
+
+  doCheck = false;
```

Comment:
@ThanePatrol did not add this in https://github.com/NixOS/nixpkgs/pull/340773

Is it needed? If so, could you add a brief comment why the tests fail?

Reply:
I'll double check this. I had an issue originally so changed this to try fix it, though that might not have been the cause.